azure-communication-ui/azure-communication-ui/src/calling/java/com/azure/android/communication/ui/calling/models/CallCompositeLocalOptions.java

```diff
@@ -65,15 +65,27 @@ public CallCompositeParticipantViewData getParticipantViewData() {
         return participantViewData;
     }
 
+    /**
+     * Set a {@linnk CallCompositeParticipantViewData} to be used
```

Comment:
@linnk - typo?

Reply:
Fixed in [1e0f384](https://github.com/Azure/communication-ui-library-android/pull/393/commits/1e0f384de2c080069e8ceaf6774b015ceae75b22)